src/components/index.js

```diff
@@ -0,0 +1,3 @@
+import App from './App/App'
+
+export default App
```

Comment:
There's no need to export each component in this file. Experience tells me that this is not easily maintainable, it grows too quickly. It's better to import each component individually.